keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -130,27 +132,36 @@ type Webhook struct {
 ### Scope
 
 Current webhook Rules applies to objects of all scopes. That means a Rule can use wildcards 
-to target both namespaced and cluster scoped objects. The proposal is to add a scope field 
+to target both namespaced and cluster-scoped objects. The proposal is to add a scope field 
```

Comment:
The text doesn't directly state why is this a useful and/or necessary change? (fix in a follow-up is fine)

Reply:
added motivation